bindings/python/docs/source/supported_types.rst

```diff
@@ -6,6 +6,11 @@ Supported Types
 PyMongoArrow currently supports a small subset of all BSON types.
 Support for additional types will be added in subsequent releases.
 
+.. note:: PyMongoArrow does not currently support extension types with Pandas/NumPy or Arrow. However, they can be used in schemas.
+   This means that ObjectId and Decimal128 are not supported in Pandas DataFrames or Arrow Tables.
```

Comment:
"not fully supported" here as well?

Reply:
ObjectId and Decimal128 are not supported in Pandas, only "bytes" and "str". This distinction is important because it will dictate how operations are applied to that column.

Reply:
What I mean is that you can still use those types in your schemas and documents, they just have limitations.

Reply:
Ah, I see what you mean. Changed.